source/rainerscript/configuration_objects.rst

```diff
@@ -3,6 +3,25 @@ configuration objects
 
 Note: configuration object parameters are case-insensitive.
 
+All configuration objects have a parameter "config.enable".
+For auto-generated configs, it is useful to have the ability to disable some
+config constructs even though they may be specified inside the config. This
+can be done via the ``config.disable`` parameter.
+If set to ``on`` or not specified, the construct will be
+used, if set to any other value, it will be ignored. This can be used
+together with the backtick functionality to configure enable and disable
+from either a file or environment variable.
+
+Example:
+
+Let's say we want to conditionally load a module. Environment variable
+``LOAD_IMPTCP`` will be either unset or ``off`` .
+Then we can use this config construct::
+
+    module(load="imptcp" config.enabled=`echo $LOAD_IMPTCP`)
+
+It the variable is set to ```off```, the module will **not** be loaded.
```

Comment:
Typo:

```diff
-It
+If
```

> `module(load="imptcp" config.enabled=`echo $LOAD_IMPTCP`)`
>  the variable is set to ```off```, the module will **not** be loaded.

and if `config.enabled` is *not* explicitly set to `off` (either explicitly on or not specified) then the module is loaded?